grc/core/io/yaml.py

```diff
@@ -39,6 +40,8 @@ def represent_ordered_mapping(self, data):
             self.represented_objects[self.alias_key] = node
 
         for item_key, item_value in six.iteritems(data):
+            if type(item_value) == attributed_str:
+                item_value = str(attributed_str)
```

Comment:
The whole point of this class is to make custom types dump-able. Instead of converting, I suggest to add a dumper for attributed_str. This would not only solve this issue, but also any conversion to yaml that might happen somewhere else or in future code...